README.md

```diff
@@ -1,10 +1,14 @@
 # url_shortener
-This app  reduce long urls . Built in nodejs, express, postgres and sequelize as orm. 
```

Comment:
Pay attention to details such as whitespaces when you write text. Also, use uppercases for names.